cmd/skopeo/main.go

```diff
@@ -32,6 +32,10 @@ func createApp() *cli.App {
 			Name:  "debug",
 			Usage: "enable debug output",
 		},
+		cli.BoolFlag{
+			Name:  "policy-insecure-allow-anything",
+			Usage: "run the tool without any policy check",
+		},
```

Comment:
(One of these days we should decide on the order of these options, i.e. their order in `--help`. Alphabetical? By topic?  But not today, this is not a reason to change this PR.)